<article class="bugdoc">
    <h2 class="bugdoc-title" id="Bug-Doc:PE22534">PE22534: E0118</h2>
    <h3>E0118</h3>
    <section id="tip">
     <h4 style="border-top: 0pt; padding: 0pt 4pt 4pt 4pt;">Bug info</h4>
                <dl>
                    <dt>BugTypeID</dt>
                    <dd>PE22534</dd>
                    <dt>Tags</dt>
                    <dd>flake8</dd>
                    <dt>Category</dt>
                    <dd>Code style and efficiency</dd>
                    <dt>Authors</dt>
                    <dd>pinpoint</dd>
                </dl>
    <h2>Reason</h2>

<pre><code>Name %r is used prior to global declaration

Emitted when a name is used prior a global declaration, which results in an error since Python 3.6. This message can't be emitted when using Python &lt; 3.6.

</code></pre>
<h4>References</h4>
<section class="footnotes">
<ol>
<li id="fn-1"><p>https://www.python.org/dev/peps/pep-0008/<a href="#fnref-1" class="footnote">&#8617;</a></p></li>
</ol>
</section>

    </section>
</article>
